### PR TITLE
Another PR template tweak

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by lyrixx — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
-| BC breaks?    | yes/no
+| BC breaks?    | no     <!-- see https://symfony.com/bc -->
 | Deprecations? | yes/no <!-- don't forget to update UPGRADE-*.md files -->
-| Tests pass?   | yes/no
-| Fixed tickets | #... <!-- #-prefixed issue number(s), if any -->
+| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
+| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
 | License       | MIT
-| Doc PR        | symfony/symfony-docs#... <!--highly recommended for new features-->
+| Doc PR        | symfony/symfony-docs#... <!-- will be required for new features -->
```
Comment (fabpot) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would say "is required for new features"

**Review by fabpot — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (nicolas-grekas) [reply .github/PULL_REQUEST_TEMPLATE.md]: updated

**Review by Simperfit — APPROVED** [review]

**Comment by Taluu** [comment]
Shouldn't the template be split into two (one for bug fixes, one for new features), as described [here](https://github.com/blog/2495-multiple-issue-and-pull-request-templates) ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
-| Fixed tickets | #... <!-- #-prefixed issue number(s), if any -->
+| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
+| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
 | License       | MIT
-| Doc PR        | symfony/symfony-docs#... <!--highly recommended for new features-->
+| Doc PR        | symfony/symfony-docs#... <!-- required for new features -->
 
 <!--
-Write a short README entry for your feature here (replace this comment block.)
+Write a short README entry for your feature/bug here (replace this comment block.)
```
Comment (phansys) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should be "feature/fix".

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Needs work.

**Comment by stof** [comment]
@Taluu most of the template is common. And when creating a PR using the github buttons, you always get the default PR template. Additional PR templates are used only when the URL contains a special query string (requiring to provide special links). I don't think we would benefit from the multiple templates here

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
-| BC breaks?    | yes/no
+| BC breaks?    | no     <!-- see https://symfony.com/bc -->
 | Deprecations? | yes/no <!-- don't forget to update UPGRADE-*.md files -->
-| Tests pass?   | yes/no
-| Fixed tickets | #... <!-- #-prefixed issue number(s), if any -->
+| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
+| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
 | License       | MIT
-| Doc PR        | symfony/symfony-docs#... <!--highly recommended for new features-->
+| Doc PR        | symfony/symfony-docs#... <!-- required for new features -->
```
Comment (ostrolucky) [inline .github/PULL_REQUEST_TEMPLATE.md]: This isn't realistic. Do you want to force users to write docs for ever little new feature (like mine here https://github.com/symfony/symfony/pull/26218/files)?

Comment (nicolas-grekas) [reply .github/PULL_REQUEST_TEMPLATE.md]: this is only a template, "force" is a bit too strong in your sentence :)
but yes, we want to ensure contributors wonder seriously about the doc PR
then, common sense will drive reality, as usual :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (nicolas-grekas) [reply .github/PULL_REQUEST_TEMPLATE.md]: fixed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (javiereguiluz) [reply .github/PULL_REQUEST_TEMPLATE.md]: @ostrolucky I agree: it's not realistic to ask that code contributors write the full docs of their features (besides, most developers can't write good docs). But we'll no longer accept pull requests with empty or very vague descriptions.

Instead, pull requests must explain (very briefly) what they do or improve or change. Also, when applicable, we'd need some simple code or config example for the feature. With that information, members of the Doc Team can write the definitive docs.

Comment (stof) [reply .github/PULL_REQUEST_TEMPLATE.md]: and if it is not the documentation PR, most features at least need a doc *issue* to make the doc team aware of the need to document this

**Review by phansys — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**